### DISCUSSION TITLE
Exclude / include all items in schema compare

### DISCUSSION THREAD

**Discussion by pauldj54** [microsoft/azuredatastudio#20629] [Ideas]
We are maintaining a list of schemata of an Azure SQL Database, but some other schemata is not maintained by us.

When we do schema compare using the project and the target database, all changes pop up in the comparison results.

Ideally, we would like to exclude individual schemata from the comparison. Since this is not possible, I would like to have the option, like in Visual Studio, to unselect everything at once (if the objects were organized by new/changes/delete would be even better).

At the moment we need to go through every object and uncheck it manually, which is almost impossible due to the dependencies between these objects.

I tried to hack the .scmp file, adding manually the exclusions, which is somehow working but it is difficult to maintain.  

Any chance to get this feature implemented?

BR. 
Paul Hernandez

**Comment by mappao** [comment]
Moreover I'd like to have an option in settings to include/exclude by default on initial comparison

**Comment by chest-r** [comment]
agreed with original.  and another enhancement to this would be to filter on action and tick/untick all. So you could exclude all deletes, or only select to include Add as an example.
